pkgs/by-name/mi/microfetch/package.nix

```diff
@@ -0,0 +1,26 @@
+{
+  lib,
+  rustPlatform,
+  fetchFromGitHub,
+}:
+rustPlatform.buildRustPackage rec {
+  pname = "microfetch";
+  version = "0.3.3";
+
+  src = fetchFromGitHub {
+    owner = "NotAShelf";
+    repo = "microfetch";
+    rev = "refs/tags/v${version}";
+    hash = "sha256-7pCy8x0HiQeuNZ3/0w4CyBkcrwn+xSuZBmrU3fZIC/c=";
```

Comment:
```suggestion
    hash = "sha256-iBfnui7xrx31euYnQxoJf1xXFYFiJnDMadLRAiRCN04=";
```

Reply:
I addressed all issues, thank you for reviewing.